python/aitemplate/backend/cuda/gemm_universal/gemm_rrr_bias.py

```diff
@@ -0,0 +1,327 @@
+"""
+)
+
+
+# in case of TMA epilogue schedule, use the transposed problem to pass the
+# column-major bias vector through the bias + elementwise epilogue (not residual)
+PROFILER_PROBLEM_ARGS_TEMPLATE_CUTLASS_3X = jinja2.Template(
+    """
+    cutlass::gemm::GemmUniversalMode::kGemm,                     // GemmUniversalMode mode
+{% if has_tma_epilogue %}
+    {
+        static_cast<coord_t>(N),
+        static_cast<coord_t>(M),
+        static_cast<coord_t>(K),
+        static_cast<coord_t>(1)
+    },                                                           // ProblemShape problem_shape
+    ({{elem_input_type}}*)(b_ptr),                               // ElementA const* ptr_A
+    {K, cute::Int<1>{}, cute::Int<0>{}},                         // StrideA dA
+    ({{elem_input_type}}*)(a_ptr),                               // ElementB const* ptr_B
+    {N, cute::Int<1>{}, cute::Int<0>{}},                         // StrideB dB
+    {
+        {ElementComputeEpilogue(1), ElementComputeEpilogue(0)},  // typename ThreadEpilogueOp::Params thread
+        nullptr,                                                 // ElementC const* ptr_C
+        {cute::Int<1>{}, cute::Int<0>{}, cute::Int<0>{}},        // StrideC dC
+        ({{elem_output_type}}*)(c_ptr) + output_offset,          // ElementD const* ptr_D
+        {cute::Int<1>{}, output_stride, cute::Int<0>{}},         // StrideD dD
+        ({{elem_input_type}}*)(bias_ptr),                        // ElementBias const* ptr_Bias
+    },                                                           // EpilogueArguments epilogue
+{% else %}
+    {
+        static_cast<coord_t>(M),
+        static_cast<coord_t>(N),
+        static_cast<coord_t>(K),
+        static_cast<coord_t>(1)
+    },                                                           // ProblemShape problem_shape
+    ({{elem_input_type}}*)(a_ptr),                               // ElementA const* ptr_A
+    {K, cute::Int<1>{}, cute::Int<0>{}},                         // StrideA dA
+    ({{elem_input_type}}*)(b_ptr),                               // ElementB const* ptr_B
+    {N, cute::Int<1>{}, cute::Int<0>{}},                         // StrideB dB
+    {
+        {ElementComputeEpilogue(1), ElementComputeEpilogue(1)},  // typename ThreadEpilogueOp::Params thread
+        ({{elem_input_type}}*)(bias_ptr),                        // ElementC const* ptr_C
+        {cute::Int<0>{}, cute::Int<1>{}, cute::Int<0>{}},        // StrideC dC
+        ({{elem_output_type}}*)(c_ptr) + output_offset,          // ElementD const* ptr_D
+        {output_stride, cute::Int<1>{}, cute::Int<0>{}},         // StrideD dD
+    },                                                           // EpilogueArguments epilogue
+{% endif %}
+"""
+)
+
+
+@registry.reg("cuda.gemm_rrr_bias.config")
+def gemm_rrr_config(func_attrs, dtype="float16"):
+    common.make_fproc(func_attrs, RRR, include_cutlass_3x_ops=True)
```

Comment:
For simplicity, I suggest we omit all the CUTLASS 3.x-related code at this point (also, as we currently can't properly test it in CircleCI). This includes:

1. Remove `include_cutlass_3x_ops=True` in this line.
2. Remove the rest of this function's body below this line.
3. Remove `PROBLEM_ARGS_TEMPLATE_CUTLASS_3X` and `PROFILER_PROBLEM_ARGS_TEMPLATE_CUTLASS_3X` above.
4. Don't render / pass the CUTLASS 3.x-related templates in the `gen_profiler` and `gen_function` functions below.

Reply:
OK, work in progress.